sys/vane/jael.hoon

```diff
@@ -2090,9 +2094,34 @@
     =+  dif=(event-log-to-hull-diff log)
     ?~  dif  +>.$
     (put-change cuz %hull u.dif)
+  ::                                                    ::  ++restore-block:et
+  ++  restore-block                                     ::  rewind before block
+    |=  block=@ud
+    ^+  +>
+    =/  old-qeu  snaps.sap
+    =:  snaps.sap       ~
+        count.sap       0
+        last-block.sap  0
+      ==
```

Comment:
Your closing `==` are consistently indented (here and in other uses of `=:`). I'm used to placing/seeing them on the same indentation level as the `=:`.
I notice that for `:*`, you don't indent the `==`. Is the indent a way of saying "this is not a leaf hoon"?

Reply:
I've seen it both ways in the past, don't know what's accepted practice nowadays.  To me, `==` or `--` followed by another line on the same indentation level feels wrong.  As you say, the variable-arity portion of `:*`, `:~`, `?-`, `?+`, `|%`, etc is at the end of the hoon while it's not for `=:`.

Reply:
Looks like hoon.hoon and arvo use indent-style, zuse.hoon uses dedent-style.

Reply:
Interesting. Indenting closing glyphs for non-leaf hoons seems sanest to me, so do carry on.